ui/js/controllers/logviewer.js

```diff
@@ -214,7 +212,7 @@ logViewerApp.controller('LogviewerCtrl', [
 
             TextLogStepModel.get($scope.job_id).then((textLogSteps) => {
                 let shouldPost = true;
-                const allErrors = _.flatten(textLogSteps.map(s => s.errors));
+                const allErrors = (textLogSteps.map(s => s.errors)).reduce((a, b) => a.concat(b), []);
```

Comment:
Believe it or not, ends up the spread syntax is faster than using ``concat`` (at least from what I can find searching around and running a simple benchmark).

Would you modify these to be something like:

    .reduce((a, b) => [...a, ...b], []);

Reply:
Wow I wasn't aware of this! Switching to the spread syntax right now.